### ISSUE TITLE
Wrong error 'no-redeclare' in Node 4.2 and 0.12

### ISSUE THREAD

**Issue by tleunen** [babel/babel-eslint#210]
This code is failing now with Node 4.2 and 0.12. Any reason to that?

```
  5:12  error  "otherProps" is already defined  no-redeclare
  5:12  error  "otherProps" is already defined  no-redeclare
```

```js
const props = {
    yo: 'yo'
};
const { ...otherProps } = props;
```

**Comment by hzoo** [comment]
Sounds like the escope upgrade issue - should be fixed by https://github.com/babel/babel-eslint/pull/209 (check the referenced issues)

**Comment by tleunen** [comment]
Any update on this?
escope has been updated, I can see the new version (3.3.0) is installed and used but still get the same error.
@Constellation, does your fix in #209 is needed to fix it?

**Comment by b2whats** [comment]
I have the same error
node -v = 0.10.32

**Comment by AlesJiranek** [comment]
same here - 0.12.7

**Comment by hzoo** [comment]
Should be fixed in the next release

**Comment by BerkeleyTrue** [comment]
Can this be release before the babel6 upgrade?

**Comment by hzoo** [comment]
Yeah already released 4.1.6

**Comment by henriquea** [comment]
Nice one! Thanks
